rke/k8s_service_options.go

```diff
@@ -124,7 +124,7 @@ func loadK8sVersionServiceOptions() map[string]v3.KubernetesServicesOptions {
 			Kubeproxy:      getKubeProxyOptions(),
 			Scheduler:      getSchedulerOptions(),
 		},
-		"v1.15.12-rancher1-1": {
+		"v1.15.12-rancher2-3": {
```

Comment:
Why are we updating `v1.15.12-rancher1-1`? Why aren't they being added for the other k8s versions, v1.16, v1.17?